x/auth/vesting/handler_test.go

```diff
@@ -91,6 +91,52 @@ func (suite *HandlerTestSuite) TestMsgCreateVestingAccount() {
 	}
 }
 
+func (suite *HandlerTestSuite) TestMsgCreatePeriodicVestingAccount() {
+	ctx := suite.app.BaseApp.NewContext(false, tmproto.Header{Height: suite.app.LastBlockHeight() + 1})
+
+	balances := sdk.NewCoins(sdk.NewInt64Coin("test", 1000))
+	addr1 := sdk.AccAddress([]byte("addr1_______________"))
+	addr3 := sdk.AccAddress([]byte("addr3_______________"))
+
+	acc1 := suite.app.AccountKeeper.NewAccountWithAddress(ctx, addr1)
+
+	period := []types.Period{{Length: 5000, Amount: balances}}
+	suite.app.AccountKeeper.SetAccount(ctx, acc1)
+	suite.Require().NoError(suite.app.BankKeeper.SetBalances(ctx, addr1, balances))
+
+	testCases := []struct {
+		name      string
+		msg       *types.MsgCreatePeriodicVestingAccount
+		expectErr bool
+	}{
+		{
+			name:      "continuous vesting account already exists",
+			msg:       types.NewMsgCreatePeriodicVestingAccount(addr1, addr3, period),
+			expectErr: true,
```

Comment:
I don't think this will actually work, as we haven't created a vesting account yet?

Can we add a passing test above? Then this one should error correctly.  Also we should change the name from "continuous vesting" to "periodic vesting".